README.md

```diff
@@ -1,2 +1,181 @@
 # My-Catalog-Ruby-Capstone
+
+<!-- describe the purpose of this project -->
+
 A console app to help you keep a record of different things you own such as books, music, albums, movies
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
+
+<!-- Table of Contents -->
+# 📗 Table of Contents
+
+- [My-Catalog-Ruby-Capstone](#my-catalog-ruby-capstone)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 My-Catalog-Ruby-Capstone ](#-my-catalog-ruby-capstone-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [Getting Started ](#getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run](#run)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+<!-- - [📝 License](#license) -->
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 My-Catalog-Ruby-Capstone <a name="about-project"></a>
+
+**# My-Catalog-Ruby-Capstone**.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server Side</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">RUBY</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.json.org/">JSON</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **List all books.**
+- **List all music albums.**
+- **List all movies.**
+- **List all genres.**
+- **List all labels.**
+- **List all authors.**
+- **Add a book.**
+- **Add a music album.**
+- **Add a movie.**
+- **Preserve data in JSON files**
+- **Database for the app**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [updated upon project completion.]()
+
```

Comment:
The following basic requirements are required for your project to proceed to the next phase of your capstone 😄 

-  you should record a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
-  All the members in the video should appear in the video and talk about the project.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
Thanks for letting us know @alexander16108 we were so focused on the completion of the project that we totally overlooked this requirement, we'll add it in no time